node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -455,16 +649,43 @@ mod tests {
             Some(b"Upgrade installer bytes".to_vec())
         );
         assert!(upgrade0.upgrade_installer_args.is_none());
-        assert_eq!(upgrade0.new_costs.unwrap().regular, 24);
-        assert_eq!(upgrade0.new_costs.unwrap().div, 25);
-        assert_eq!(upgrade0.new_costs.unwrap().mul, 26);
-        assert_eq!(upgrade0.new_costs.unwrap().mem, 27);
-        assert_eq!(upgrade0.new_costs.unwrap().initial_mem, 28);
-        assert_eq!(upgrade0.new_costs.unwrap().grow_mem, 29);
-        assert_eq!(upgrade0.new_costs.unwrap().memcpy, 30);
-        assert_eq!(upgrade0.new_costs.unwrap().max_stack_height, 31);
-        assert_eq!(upgrade0.new_costs.unwrap().opcodes_mul, 32);
-        assert_eq!(upgrade0.new_costs.unwrap().opcodes_div, 33);
+        assert_eq!(
+            upgrade0.new_wasm_config.as_ref().unwrap().opcode_costs,
+            OpCodeCosts {
```

Comment:
Let's construct this value outside of the assertion and bind it to a name like `expected_costs`.